RELEASE-NOTES.rst

```diff
@@ -18,6 +18,176 @@
 Release notes
 =============
 
+v0.11.0
+-------
+
+User interface
+--------------
+
+-  Updates schema of forms to use the new sorted select menu.
+-  Displays new metadata fields: ``seriesStatement`` and ``partOf``:
+
+   -  Displays fields in detailed view and in editor.
+   -  Removes ``partOf`` field from brief view.
+   -  Adds ``oneOf`` attribute in order to link issuance ``maintype``
+      and ``subtype`` in editor.
+
+-  Adds missing translations of item notes types.
+-  Limits length of document title to 150 characters in detailed view.
+   Adds a *Show more*/*Show less* link if the title is truncated.
+-  Sorts pickup locations alphabetically when placing a request for an
+   item.
+
+Professional interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  Adds inventory list functionnality and view based on ``item``
+   resources. The can acces them using the ``Admin`` menu. This
```

Comment:
Line 45: please change 
The can acces them using the ``Admin`` menu.
by
The librarian can acces them using the ``Admin`` menu.